src/test/java/com/revaturelabs/ask/AskApplicationControllerTests.java

```diff
@@ -1,984 +1,991 @@
+// public void testGetImagesForResponseStatusException() {
+//
+// when(questionServiceMock.getById(1)).thenThrow(QuestionNotFoundException.class);
+// assertEquals(ResponseEntity.ok(null), questionControllerImpl.getImages(1));
+// }
+//
+// @Test
+// public void testGetResponses() {
+//
+// Question question = new Question();
+//
+// when(questionServiceMock.getById(1)).thenReturn(question);
+// assertEquals(ResponseEntity.ok(null), questionControllerImpl.getResponses(1));
+// }
+//
+// @Test(expected = ResponseStatusException.class)
+// public void testGetResponsesForResponseStatusException() {
+//
+// when(questionServiceMock.getById(1)).thenThrow(QuestionNotFoundException.class);
+// assertEquals(ResponseEntity.ok(null), questionControllerImpl.getResponses(1));
+// }
+//
+// @Test
+// public void testGetAllQuestions() {
+//
+// List<Question> questions = new ArrayList<Question>();
+// Page<Question> page = new PageImpl<>(questions);
+// when(questionServiceMock.getAll(0, 20)).thenReturn(page);
+// assertEquals(ResponseEntity.ok(page.getContent()), questionControllerImpl.getAllQuestions(0,
+// 20));
+// assertEquals(ResponseEntity.ok(page.getContent()), questionControllerImpl.getAllQuestions(null,
+// null));
+// }
+//
+// @Test
+// public void testHighlightResponse() {
+//
+// Question question = new Question();
+//
+// when(questionServiceMock.highlightResponse(1, 1)).thenReturn(question);
+// assertEquals(ResponseEntity.ok(questionControllerImpl.highlightResponse(1, 1).getBody()),
+// questionControllerImpl.highlightResponse(1, 1));
+// }
+//
+// @Test(expected = ResponseStatusException.class)
+// public void testHighlightResponseForQuestionNotFoundException() {
+//
+// when(questionServiceMock.highlightResponse(1, 1)).thenThrow(QuestionNotFoundException.class);
+// assertEquals(ResponseEntity.ok(questionControllerImpl.highlightResponse(1, 1).getBody()),
+// questionControllerImpl.highlightResponse(1, 1));
+// }
+//
+// @Test(expected = ResponseStatusException.class)
+// public void testHighlightResponseForQuestionConflictException() {
+//
+// when(questionServiceMock.highlightResponse(1, 1)).thenThrow(QuestionConflictException.class);
+// assertEquals(ResponseEntity.ok(questionControllerImpl.highlightResponse(1, 1).getBody()),
+// questionControllerImpl.highlightResponse(1, 1));
+// }
+//
+// @Test(expected = ResponseStatusException.class)
+// public void testHighlightResponseFor() {
+//
+// when(questionServiceMock.highlightResponse(1,
+// 1)).thenThrow(DataIntegrityViolationException.class);
+// assertEquals(ResponseEntity.ok(questionControllerImpl.highlightResponse(1, 1).getBody()),
+// questionControllerImpl.highlightResponse(1, 1));
+// }
+//
+// @Test
+// public void testFilterByTags() {
+//
+// List<Question> questions = new ArrayList<Question>();
+// Page<Question> page = new PageImpl<>(questions);
+// Stream<Question> question = page.get();
+// List<String> tags = new ArrayList<String>()
```

Comment:
Why not just remove this file now instead of commenting out all of the code? In fact, if you've commented out all of the code in a file, just remove it and make your pull request again.

Reply:
Will there be no merge conflicts if I remove these files?

Reply:
@jgalarza101 , these files can't stay. There's no reason for them to be there. If you get merge conflicts, you'll have to fix them.

Reply:
I deleted the unnecessary test files and pushed. Hopefully, there are no merge conflicts.